imgui/core.pyx

```diff
@@ -1,6 +1,3 @@
-# distutils: language = c++
```

Comment:
The source files were already set up from setup.py when not using cython, I made the behavior consistent by not defining any sources here. This allows providing only the source files you want.
The include directories were already provided through the Extension